app/models/shipit/unlimited_api_client.rb

```diff
@@ -0,0 +1,10 @@
+module Shipit
+  class UnlimitedApiClient
```

Comment:
Yes, I was tired of crafting queries. It's optionally enabled with an environment variable and disabled by default.